packages/amplify-dotnet-function-template-provider/src/utils/dynamoDBWalkthrough.ts

```diff
@@ -85,7 +85,7 @@ export async function getTableParameters(context: any, dynamoAnswers: any): Prom
   } // Looking for table parameters on local configuration
   const projectBackendDirPath = context.amplify.pathManager.getBackendDirPath();
   const resourceDirPath = path.join(projectBackendDirPath, 'storage', dynamoAnswers.resourceName);
-  const parametersFilePath = path.join(resourceDirPath, 'parameters.json');
+  const parametersFilePath = path.join(resourceDirPath, 'build', 'parameters.json');
```

Comment:
this miss showed up in new test coverage.

Reply:
Line 86 to 96 can be simplified down to a single stateManager call. Technically, the try/catch handles additional error scenarios, like if permissions is denied reading the file, but I think we should throw in that case.

```ts
return stateManager.getResourceParametersJson(undefined, AmplifyCategories.STORAGE, dynamoAnswers.resourceName, { throwIfNotExists: false, default: {} });
```

Reply:
@jhockett done.